app/models/alma_pod_records/alma_pod_sender.rb

```diff
@@ -26,10 +27,21 @@ def send
     def parameters
       {
         'upload[name]': @filename,
-        'upload[files][]': UploadIO.new(File.new(@filename), 'application/marcxml+xml')
+        'upload[files][]': UploadIO.new(File.new(@filename), 'application/marcxml+xml'),
+        'stream': stream
       }
     end
 
+    def stream
+      if @compressed
+        LibJobs.config[:pod_test_stream]
+        # 'princeton-test-set'
```

Comment:
I dont think the comments are needed since. Someone can see https://github.com/pulibrary/lib_jobs/pull/409/files#diff-cb047af3dd43afd8e756ef9289e96ce3139a3f7e5fba09e05d8859953257587cR16 what they represent.